db/migrate/20150320160909_add_send_build_merge_email.rb

```diff
@@ -0,0 +1,9 @@
+class AddSendBuildMergeEmails < ActiveRecord::Migration
+  def up
```

Comment:
This works, but I'd just define a `change` method rather than explictly defining the up and down.

```
  def change
    add_column :repositories, :send_build_merge_emails, :boolean, default: true, null: false
  end
```